test/shell_runner.go

```diff
@@ -133,6 +140,9 @@ func (runner *ShellRunner) RunWith(opts command.Options, cmd string, args ...str
 	// run the command inside the custom environment
 	result, err = command.RunWith(opts, cmd, args...)
 	if Debug {
+		if err != nil {
+			fmt.Printf("ERROR: %v\n", err)
+		}
```

Comment:
Thoughts on printing this after the command and it's output?

Reply:
Good idea, done